plugin/src/main/java/org/opensearch/ml/action/models/DeleteModelTransportAction.java

```diff
@@ -109,7 +114,7 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<Delete
                                 if (!access) {
                                     actionListener
                                         .onFailure(
-                                            new MLValidationException("User Doesn't have privilege to perform this operation on this model")
+                                            new MLValidationException("User doesn't have privilege to perform this operation on this model")
```

Comment:
instead of `this operation`, can we say: `delete operation`?

Reply:
Yeah will work with tech writer to get appropriate error messages